### PR TITLE
feat: Add average cpu utilization and cpu busy in harvest dashboards

### PR DESCRIPTION
**Pull request by rahulguptajss**
Fixes#1842

**CDot:**

Before:
image

After:
image

**Node:**

Before:
image

After:
image

**Cluster:**
Before:
image

After:
image

Before:
image

After:
image

**Metro Cluster:**

Before:
image

After:
image

### CODE REVIEW

File: grafana/dashboards/cmode/mcc_cluster.json [hunk 1]
```diff
@@ -2656,7 +2657,7 @@
           "text": "5",
           "value": "5"
         },
-        "datasource": "${DS_PROMETHEUS}",
+        "datasource": "Prometheus",
```
Comment (Hardikl) [inline grafana/dashboards/cmode/mcc_cluster.json:2660]: {DS_PROMETHEUS} ?

**Review by Hardikl — APPROVED** [review]

**Review by Hardikl — DISMISSED** [review]

File: grafana/dashboards/cmode/cdot.json [hunk 2]
```diff
@@ -465,6 +452,100 @@
           "title": "Top $TopResources Clusters by Average CPU Utilization",
           "type": "timeseries"
         },
+        {
+          "datasource": "${DS_PROMETHEUS}",
+          "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/cdot.json:457]: is this clearer?
> System CPU resource utilization is a computed percentage that indicates how busy the system is based on a combination of the most heavily utilized domain and idle CPU cycles. This metric determines the amount of available CPU and will vary based on domains reaching their limits OR exhausting all idle CPU cycles, whichever comes first.

File: grafana/dashboards/cmode/cluster.json [hunk 3]
```diff
@@ -1325,7 +1425,7 @@
         },
         {
           "datasource": "${DS_PROMETHEUS}",
-          "description": "",
+          "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/cluster.json:1428]: change to match cdot dashboard

File: grafana/dashboards/cmode/cluster.json [hunk 4]
```diff
@@ -1876,6 +2050,7 @@
         },
         {
           "datasource": "${DS_PROMETHEUS}",
+          "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/cluster.json:2053]: change to match cdot dashboard
might be worth pulling this description into a constant var and updating panels to use it

File: grafana/dashboards/cmode/cluster.json [hunk 5]
```diff
@@ -2524,6 +2785,99 @@
           "timeShift": null,
           "title": "Average CPU Utilization",
           "type": "timeseries"
+        },
+        {
+          "datasource": "${DS_PROMETHEUS}",
+          "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/cluster.json:2791]: change to match cdot dashboard

File: grafana/dashboards/cmode/node.json [hunk 6]
```diff
@@ -420,10 +420,12 @@
     },
     {
       "datasource": "${DS_PROMETHEUS}",
-      "description": "",
+      "description": "System CPU resource utilization. Returns a computed percentage for the default CPU field. Basically computes a 'cpu usage summary' value which indicates how 'busy' the system is based upon the most heavily utilized domain. The idea is to determine the amount of available CPU until we're limited by either a domain maxing out OR we exhaust all available idle CPU cycles, whichever occurs first.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/node.json:423]: change to match cdot dashboard

**Review by cgrinds — APPROVED** [review]
nice job. one minor change

**Review by cgrinds — APPROVED** [review]

**Pull request merged**